OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.md

```diff
@@ -0,0 +1,241 @@
+
+These errors may happen if the `hid_uclogic` kernel module is loaded at the same time as OpenTabletDriver.
+
+You can remove and blacklist this module to stop it from loading in the future by running the following commands:
+```bash
+echo "blacklist hid_uclogic" | sudo tee -a /etc/modprobe.d/blacklist.conf
+sudo rmmod hid_uclogic
+```
+
+##### Missing udev rules {#udev}
+
+> Not permitted to open HID class device at /dev/hidrawX
+
+If you have followed the previous instructions, usually this is the only reason OpenTabletDriver will fail to detect your tablet when it's supported.
+
+The issue is resolved by making sure that you have proper udev rules set up.
+
+###### From Package Manager
+
+If you installed OpenTabletDriver via your package manager, you may just need to reload and trigger the rules:
+```bash
+sudo udevadm control --reload-rules
+sudo udevadm trigger
+```
+
+If it still doesn't work after this point, and you installed OpenTabletDriver with a package manager, please reboot, some systems may require this.
+
+###### Portable install
+
+If you installed OpenTabletDriver without using a package manager you will not have proper udev rules set up and must build your own.
+It is highly recommended that you use your package manager where available.
+
+This process will require you to have the dotnet 7 SDK installed before proceeding:
+```bash
+# Clone the repository, change current directory to the repository
+git clone https://github.com/OpenTabletDriver/OpenTabletDriver.git
+cd ./OpenTabletDriver
+
+# Generate rules, moves them to the udev rules directory
+./generate-rules.sh
+sudo mv ./bin/99-opentabletdriver.rules /etc/udev/rules.d/99-opentabletdriver.rules
+
+# Reload udev rules
+sudo udevadm control --reload-rules
+
+# Clean up leftovers
+cd ..
+rm -rf OpenTabletDriver
+```
+Then restart OpenTabletDriver and replug your tablet.
+
+#### It is not a supported tablet, what can I do about that?
+
+If you want to add support for your tablet on your own, feel free to look at a similar configuration on our GitHub and our configuration documentation.
+
+If you would like help with supporting your tablet, or would prefer us to do it, join our [Discord](/Discord) and create a post in `#config-creation` or a support channel.
+
+### My cursor is teleporting? {#ghost-cursor}
+
+This occurs when another program is reading the tablet at the same time as OpenTabletDriver, if this is happening everywhere on the system,
+the solution to this is very simple:
+```bash
+echo "blacklist wacom" | sudo tee -a /etc/modprobe.d/blacklist.conf
+sudo rmmod wacom
+```
+If you are using a Wayland compositor, it may be required for you use artist mode to pass the tablet as a virtual tablet
+rather than an absolute mouse because certain applications like SDL and XWayland may incorrectly handle the cursor otherwise.
+This is a rather simple setup which you can find <a href="#artist-mode">here</a>.
```

Comment:
```suggestion
If you are using a Wayland compositor, it may be required for you to use artist mode to pass the tablet to the driver as a virtual tablet, rather than an absolute mouse - certain applications like SDL and XWayland may incorrectly handle the cursor otherwise.
This is a rather simple config change, which can be found <a href="#artist-mode">here</a>.
```